.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: request
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
```

Comment:
Maybe something like if its a command, what is the command, how would a user use this?
Just for more specificity since otherwise we'd have to follow up on that, we can just write it outright.

Reply:
true this is the default template that's why it's so generic, we should be more specific indeed